py3status/module.py

```diff
@@ -350,6 +350,35 @@ def set_module_options(self, module):
 
             self.i3s_module_options['separator_block_width'] = sep_block_width
 
+        for name in ('background', 'border'):
+            if name in mod_config:
+                param = mod_config[name]
+                if not isinstance(param, basestring):
+                    raise TypeError(
+                        'Invalid attribute `{}`, '
+                        'should be an int, not `{}`'
+                        ''.format(name, param)
```

Comment:
you can remove the `''`

```python
(
    'I like parenthesis {} much, '
    'they let me have such fun.'
    .format('so')
)
```

The wording is confusing it sounds like you are going to give the type, either specify the type or better just say `got ...` or similar